cluster/addons/storage-class/vsphere/default.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
   annotations:
     storageclass.kubernetes.io/is-default-class: "true"
   labels:
-    kubernetes.io/cluster-service: "true"
+    addonmanager.kubernetes.io/mode: Reconcile
```

Comment:
i think `addonmanager.kubernetes.io/mode: EnsureExists` is more appropriate for storage classes.

Reply:
Yes, you are right. Thank you for the spot.